classes/Tools.php

```diff
@@ -2308,6 +2308,7 @@ public static function generateHtaccess($path = null, $rewrite_settings = null,
         fwrite($write_fd, "AddType font/otf .otf\n");
         fwrite($write_fd, "AddType application/font-woff .woff\n");
```

Comment:
```suggestion
        fwrite($write_fd, "AddType font/woff .woff\n");
```

this also makes more sense imho